playbooks/existing-server.yml

```diff
@@ -0,0 +1,15 @@
+---
+# existing-server.yml is an advanced provisioning option that doesn't use a genesis
+# role to create a new server and instead applies Streisand to an existing
+# remote server.
+#
+# Try and detect the remote server's provider & apply required workarounds
+- include: provider-detect.yml
```

Comment:
Good catch! I agree with both you and Nick, I can include `python.yml` in `localhost.yml` & `existing-server.yml` as the first statement before `provider-detect.yml` is included.